lib/config/webpack.js

```diff
@@ -1,5 +1,7 @@
 const merge = require('merge');
 
 module.exports = {
-  rules: merge(require('./rules/webpack')),
+  rules: merge(require('./rules/webpack'), {
+    'no-process-env': 'off',
```

Comment:
Using this in webpack helps actually kill code that only applies in a particular environment.